src/tools/tidy/src/ui_tests.rs

```diff
@@ -15,7 +15,7 @@ use std::path::{Path, PathBuf};
 const ENTRY_LIMIT: u32 = 900;
 // FIXME: The following limits should be reduced eventually.
 
-const ISSUES_ENTRY_LIMIT: u32 = 1672;
+const ISSUES_ENTRY_LIMIT: u32 = 1673;
```

Comment:
Problem: please don't add the test to `ui/issues` because it's semantically meaningless, please find a more meaningful subdirectory.